integration-tests/api-gateway/env-variables.ts

```diff
@@ -1,10 +1,8 @@
-import { getSSMParameter } from "../resources/ssm-param-helper";
-import { stackOutputs } from "../resources/cloudformation-helper";
-
-type EvidenceRequest = {
+export type EvidenceRequest = {
   scoringPolicy: string;
   strengthScore: number;
 };
+
 export const CLIENT_ID = process.env.CLIENTID ?? "ipv-core-stub-aws-prod";
```

Comment:
Can we update the default value for this? I see we are overriding it in 2 places